### DISCUSSION TITLE
Startup Welcome Screen as shown in screenshots?

### DISCUSSION THREAD

**Discussion by pewf** [NvChad/NvChad#1928] [Q&A] [answered]
Hello, I'm just asking how I can get this startup screen or the configuration for it.

![Screen Shot 2023-04-15 at 10 35 47 PM](https://user-images.githubusercontent.com/78571856/232265044-6a38f863-a97f-4990-9576-42fd8f9f9537.png)

**Comment by siduck** [comment, marked as answer]
@LowSpecGamerROBLOX pls read core/default_config.lua file and our docs
